build/emoji.txt

```diff
@@ -0,0 +1,79 @@
+😁	5
```

Comment:
We should drop this file now that we are including `Emoji_Sentiment_Data_v1.0.csv`